common/static/common/js/spec/components/feedback_spec.js

```diff
@@ -137,7 +137,7 @@
         var view;
         view = new PromptView.Confirmation(this.options).show();
         expect(this.inFocusSpy).toHaveBeenCalled();
-        return ViewHelpers.verifyElementInFocus(view, ".wrapper-prompt")
+        expect(view.$(".wrapper-prompt")).toBeFocused();
```

Comment:
@muzaffaryousaf For some reason `verifyElementInFocus` is not working. So i have used `toBeFocused` from `jasmine-jquery`.